MSAL/src/MSALDeviceInformation.m

```diff
@@ -95,6 +108,11 @@ - (void) initExtraDeviceInformation:(MSIDDeviceInfo *)deviceInfo
     [_extraDeviceInformation setValue:deviceInfo.ssoExtensionMode == MSIDSSOExtensionModeFull ? @"Yes" : @"No" forKey:MSAL_DEVICE_INFORMATION_SSO_EXTENSION_FULL_MODE_KEY];
 }
 
+- (void) addRegisteredDeviceMetadataInformation:(NSDictionary *)deviceInfoMetadata
```

Comment:
nit: can you remove space after (void)?

Reply:
Removed already but its not showing here, in diff it showing.